### DISCUSSION TITLE
How to change a recipe to upload to a prefix.dev channel

### DISCUSSION THREAD

**Discussion by looooo** [prefix-dev/rattler-build#1292] [Q&A]
I want to move our build-pipelines of the development snapshots of freecad to rattler build. Also as we are forced to remove older builds from the anaconda freecad channel due to space limitation I want to upload the packages to prefix.dev. Is there any guide how to setup the feedstock to achieve this goals? Any help is welcome.

**Comment by wolfv** [comment]
That's great! 
I don't think anyone has made the changes to conda-smithy to allow this automatically. Uploading to prefix with rattler-build is fairly trivial, especially if you have trusted publishers configured (although that only works with GitHub actions right now). Otherwise we'd need to configure an API key :)

**Comment by looooo** [comment]
Thanks for your response. I have a follow up question:
I saw some information about a new tool arriving in pixi: `pixi build`
As we already added pixi as a build tool to freecad, does it makes sense to wait for pixi build to create conda-packages or should we go with feedstocks. What do you think?
